0.0.3/Readme.md

```diff
@@ -0,0 +1,116 @@
+    // This may lead to cut off labels. Therefore, implementors can decide to
+    // clean their cache when the minor version changes. Changes to the patch
+    // level MUST only have changes to tiles that are contained within one tile.
+    // When tiles change significantly, the major version MUST be increased.
+    // Implementations MUST NOT use tiles with different major versions.
+    "version": "1.0.0",
+
+    // OPTIONAL. Default: null. Contains an attribution to be displayed
+    // when the map is shown to a user. Implementations MAY decide to treat this
+    // as HTML or literal text. For security reasons, make absolutely sure that
+    // this field can't be abused as a vector for XSS or beacon tracking.
+    "attribution": "<a href='http://openstreetmap.org'>OSM contributors</a>",
+
+    // REQUIRED.
+    // An integer specifying the minimum zoom level.
+    "minzoom": 0,
+
+    // REQUIRED.
+    // An integer specifying the maximum zoom level. MUST be >= minzoom.
+    "maxzoom": 11,
+
+    // OPTIONAL.
+    // The zoom value for the quadkey index. MUST be =< maxzoom.
+    // If quadkey_zoom is > minzoom, then on each tile request from zoom between
+    // minzoom and quadkey_zoom, the tiler will merge each quadkey asset lists.
+    // The use of quadkey_zoom can be beneficial when dealing with a high number
+    // of files and a large area.
+    "quadkey_zoom": 0,
+
+    // REQUIRED. Default: [-180, -90, 180, 90].
+    // The maximum extent of available map tiles. Bounds MUST define an area
+    // covered by all zoom levels. The bounds are represented in WGS:84
+    // latitude and longitude values, in the order left, bottom, right, top.
+    // Values may be integers or floating point numbers.
+    "bounds": [ -180, -85.05112877980659, 180, 85.0511287798066 ],
+
+    // OPTIONAL. Default: null.
+    // The first value is the longitude, the second is latitude (both in
+    // WGS:84 values), the third value is the zoom level as an integer.
+    // Longitude and latitude MUST be within the specified bounds.
+    // The zoom level MUST be between minzoom and maxzoom.
+    // Implementations can use this value to set the default location. If the
+    // value is null, implementations may use their own algorithm for
+    // determining a default location.
+    "center": [ -76.275329586789, 39.153492567373, 8 ]
+
+    // REQUIRED. A dictionary of per quadkey dataset in form of {quadkeys: [datasets]} pairs.
+    // Keys MUST be valid quadkeys index with zoom level equal to mosaic `minzoom` (or `quadkey_zoom` if present).
+    // Values MUST be arrays of strings (url or sceneid) pointing to a
+    // Cloud Optimized dataset with bounds intersecting with the quadkey bounds.
+    "tiles": {
+        "030130": [
+            "s3://my-bucket/dir/file1.tif",
+            "s3://my-bucket/dir/file2.tif",
+        ]
+    },
+
+    // OPTIONAL. Default: null.
+    // A string describing the type of the asset.
+    "asset_type": "COG",
```

Comment:
TBD

Reply:
More: I see `asset_type` being either `COG` or `STAC`. This is optional and `informative`.